src/main/java/io/galeb/manager/handler/VirtualHostHandler.java

```diff
@@ -102,6 +104,21 @@ public void afterDelete(VirtualHost virtualhost) throws Exception {
         afterDelete(virtualhost, LOGGER);
     }
 
+    private void checkDupOnAliases(final VirtualHost virtualHost) throws Exception {
+        final long farmId = virtualHost.getFarmId();
+        if (farmId == -1L) return;
+        final Set<String> aliases = virtualHost.getAliases();
```

Comment:
getAliases() always returns a Set of String empty if not exist aliases at moment?